tomcat/cluster.sls

```diff
@@ -12,4 +11,4 @@ include:
     {% endif %}
     - require_in:
       - file: server_xml
-{% endif %}
+    - unless: test "`uname`" = "Darwin"
```

Comment:
@noelmcloughlin, not being a Mac guy myself I'll ask: does `uname = 'Darwin'` cover all the same cases that `grains.os != 'MacOS'` does? Isn't the latter broader?

Reply:
The outcome should be identical because of _unless_ clause. I have no idea which pattern is preferable (Jinja2 v.s. salt. comparison check).  

I usually use Jinja2, but here experimented with _unless/onlyif_ and seems okay.  According to stackoverflow most people check $OSTYPE variable, but its bash specific.  

MacOS bash:
```
:~$ test `uname` = "Darwin"
:~$ echo $?
0
```
Ubuntu bash:
```
:~$ test `uname` = "Darwin"
:~$ echo $?
1
```